src/ContentAPI.ts

```diff
@@ -33,17 +38,25 @@ export interface ContentAPI {
     deploymentOptions: DeploymentOptions<T>,
     options?: RequestOptions
   ): Promise<T[]>
+
+  /**
+   * @deprecated use iterateThroughDeployments instead
+   */
   streamAllDeployments<T extends DeploymentBase = DeploymentWithMetadataContentAndPointers>(
     deploymentOptions: DeploymentOptions<T>,
     options?: RequestOptions
   ): Readable
+  iterateThroughDeployments<T extends DeploymentBase = DeploymentWithMetadataContentAndPointers>(
+    deploymentOptions?: DeploymentOptions<T>,
+    options?: RequestOptions
+  ): AsyncIterable<T>
   downloadContent(contentHash: ContentFileHash, options?: RequestOptions): Promise<Buffer>
   isContentAvailable(cids: ContentFileHash[], options?: RequestOptions): Promise<AvailableContentResult>
-  pipeContent(
-    contentHash: ContentFileHash,
-    writeTo: ReadableStream<Uint8Array>,
-    options?: RequestOptions
-  ): Promise<Map<string, string>>
+
+  /**
+   * pipeContent only works in Node.js like environments
```

Comment:
should we deprecate this too?

Reply:
I prefer to do it in a different PR to not block the fix of this PR